pom.xml

```diff
@@ -74,7 +75,48 @@
       <groupId>com.datastax.spark</groupId>
       <artifactId>spark-cassandra-connector_${scala.main.version}</artifactId>
       <version>${connector.version}</version>
+      <exclusions>
```

Comment:
The reason we're excluding the Java Driver from the SCC dependency and adding separatly is that the version that comes with SCC (i.e. `4.13.0`) doesn't support Vectors